src/core/l10n.js

```diff
@@ -18,97 +18,17 @@ if (html && !html.hasAttribute("lang")) {
 }
 
 // We use en-US as the base
-const base = {
-  about_respec: "About",
-  abstract: "Abstract",
-  bug_tracker: "Bug tracker:",
-  close_parens: ")",
-  definition_list: "Definitions",
-  editors_note: "Editor's note",
-  feature_at_risk: "(Feature at Risk) Issue",
-  info_references: "Informative references",
-  issue_summary: "Issue Summary",
-  issue: "Issue",
-  list_of_definitions: "List of Definitions",
-  norm_references: "Normative references",
-  note: "Note",
-  open_bugs: "open bugs",
-  open_parens: "(",
-  participate: "Participate",
-  references: "References",
-  save_as: "Save as",
-  save_snapshot: "Export",
-  search_specref: "Search Specref",
-  toc: "Table of Contents",
-  warning: "Warning",
-};
+const base = {};
 
-const ko = {
-  abstract: "요약",
-};
+const ko = {};
```

Comment:
No need to keep these local variables 👀

Reply:
Was removing things incrementally... reduce -> empty -> delete